code/modules/antagonists/wizard/equipment/wizard_spellbook.dm

```diff
@@ -136,6 +136,14 @@
 		possible_entry.set_spell_info() //loads up things for the entry that require checking spell instance.
 		entries |= possible_entry
 
+/obj/item/spellbook/ui_status(mob/user, /datum/ui_state/state)
+	if(user.is_blind())
+		to_chat(user, span_warning("You are blind and can't read anything!"))
+		return UI_CLOSE
```

Comment:
Please check that Dullahan Wizards can still read their spellbook, they're funny and I wanna keep them as a possible option.

Reply:
![Wa6W0FSavv](https://github.com/tgstation/tgstation/assets/5195984/2103b2fd-efe3-41a5-8339-3ad27c4be5d9)